dot/sync/disjoint_block_set_integeration_test.go

```diff
@@ -43,14 +39,14 @@ func TestDisjointBlockSet(t *testing.T) {
 	s.addHeader(header)
 	require.True(t, s.hasBlock(header.Hash()))
 	require.Equal(t, 2, s.size())
-
 	expected = &pendingBlock{
-		hash:    header.Hash(),
-		number:  header.Number,
-		header:  header,
-		clearAt: time.Time{}.Add(ttl),
+		hash:   header.Hash(),
+		number: header.Number,
+		header: header,
 	}
-	require.Equal(t, expected, s.getBlock(header.Hash()))
+	block1 := s.getBlock(header.Hash())
+	block1.clearAt = time.Time{}
```

Comment:
we should assert the `clearAt` field somehow (even if it's not an exact assertion, at least verify it's set in the future or something)